@xen-orchestra/web/src/components/site/SiteHeader.vue

```diff
@@ -2,8 +2,8 @@
   <UiHeadBar :icon="faSatellite">
     Xen Orchestra Appliance
     <template #actions>
-      <UiButton :left-icon="faPlus" variant="secondary" accent="brand" size="medium" @click="goToNewVm()">
-        {{ t('new-vm') }}
+      <UiButton :left-icon="faPlus" variant="secondary" accent="brand" size="medium" @click="goToNewPool()">
+        {{ t('connect-pool') }}
       </UiButton>
```

Comment:
```suggestion
      <RouterLink to="/pool/connect">
        <UiButton :left-icon="faPlus" variant="secondary" accent="brand" size="medium">
          {{ t('connect-pool') }}
        </UiButton>
      </RouterLink>
```

I think you can use `RouterLink`